versions/3.1.0.md

```diff
@@ -2294,16 +2294,20 @@ Where JSON Schema indicates that behavior is defined by the application (e.g. fo
 
 ##### Properties
 
-The OpenAPI Schema Object is a JSON Schema vocabulary which extends JSON Schema Core and Validation vocabularies. As such any keyword available for those vocabularies is by definition available in OpenAPI, and will work the exact same way.
+The OpenAPI Schema Object [dialect](https://json-schema.org/draft/2019-09/json-schema-core.html#rfc.section.4.3.3) is defined as requiring the [OAS base vocabulary](#baseVocabulary), in addition to the vocabularies as specified in the JSON Schema draft 2019-09 [general purpose meta-schema](http://json-schema.org/draft/2019-09/json-schema-core.html#rfc.section.8).
+
+The OpenAPI Schema Object dialect for this version of the specification is identified by the URI `https://spec.openapis.org/oas/3.1/dialect/base` (the <a name="dialectSchemaId"></a>"OAS dialect schema id").
```

Comment:
Something that just occurred to me due to the ongoing discussion about `discriminator`, particularly if the `selectBy` idea is adopted: It might be better to put `discriminator` in its own vocabulary since it is complex to implement and (particularly if `selectBy` were added) not strictly necessary.

Putting it in its own vocabulary allows implementations to choose whether to implement it separately from the other OAS extension keywords, and could allow for dialect configurations that require the other OAS extension keywords but leave `discriminator` as optional.  idk if that's really a great idea or not but thought I would mention it.  We moved `unevaluatedProperties` and `unevaluatedItems` into their own vocabulary for similar reasons- they are complex and potentially expensive so some use cases might prefer not to deal with them (streaming implementations, for example, are much more costly with these).

Reply:
We believe this issue is one that requires further conversation but don't believe it should hold up merging the current PR that introduces the dialect property.